dask/dataframe/io/tests/test_csv.py

```diff
@@ -926,6 +926,21 @@ def test_error_if_sample_is_too_small():
                   pd.read_csv(fn, header=None, skiprows=3))
 
 
+def test_read_csv_names_not_none():
+    text = ('Alice,100\n'
+            'Bob,-200\n'
+            'Charlie,300\n'
+            'Dennis,400\n'
+            'Edith,-500\n'
+            'Frank,600\n')
+    names = ['name', 'amount']
+    with filetext(text) as fn:
+        ddf = dd.read_csv(fn, names=names, blocksize=16).compute()
+        df = pd.read_csv(fn, names=names)
+        ddf.index = range(len(ddf.index))
+        assert_eq(df, ddf)
```

Comment:
I recommend instead the `check_index=False` keyword to `assert_eq`.  That way you can pass the dask dataframe directly to the assert_eq function, which will run a number of useful checks.

Otherwise this seems great to me.  Thank you for taking this on!

Reply:
Ok, good suggestion! Don't know the codebase very well...

Reply:
Submitting PRs and going through the review process is a great way to learn!